app/models/ability.rb

```diff
@@ -92,6 +92,8 @@ def initialize(member) # rubocop:disable Metrics/AbcSize
     can :create,  Harvest
     can :update,  Harvest, owner_id: member.id
     can :destroy, Harvest, owner_id: member.id
+    can :update,  Harvest, owner_id: member.id, planting: { owner_id: member.id }
+    can :destroy, Harvest, owner_id: member.id, planting: { owner_id: member.id }
```

Comment:
AFAICT the tests cover all the cases of interest (no planting specified, planting owned by member, planting not owned by member), so I think we're good.